{"message":"error ....in create user","level":"error","timestamp":"2021-06-02T16:41:05.802Z"}
{"message":"error ....in create user","level":"error","timestamp":"2021-06-02T16:41:15.989Z"}
{"message":"error......in update handler","level":"error","timestamp":"2021-06-03T14:21:27.745Z"}
{"message":"error ....in create user","level":"error","timestamp":"2021-06-04T10:17:29.750Z"}
{"message":"db error Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/","reason":{"type":"ReplicaSetNoPrimary","setName":null,"maxSetVersion":null,"maxElectionId":null,"servers":{},"stale":false,"compatible":true,"compatibilityError":null,"logicalSessionTimeoutMinutes":null,"heartbeatFrequencyMS":10000,"localThresholdMS":15,"commonWireVersion":null},"level":"error","stack":"MongooseServerSelectionError: Could not connect to any servers in your MongoDB Atlas cluster. One common reason is that you're trying to access the database from an IP that isn't whitelisted. Make sure your current IP address is on your Atlas cluster's IP whitelist: https://docs.atlas.mongodb.com/security-whitelist/\n    at NativeConnection.Connection.openUri (C:\\Users\\LENOVO\\Desktop\\typesript_npm\\node_modules\\mongoose\\lib\\connection.js:846:32)\n    at C:\\Users\\LENOVO\\Desktop\\typesript_npm\\node_modules\\mongoose\\lib\\index.js:351:10\n    at C:\\Users\\LENOVO\\Desktop\\typesript_npm\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:31:5\n    at new Promise (<anonymous>)\n    at promiseOrCallback (C:\\Users\\LENOVO\\Desktop\\typesript_npm\\node_modules\\mongoose\\lib\\helpers\\promiseOrCallback.js:30:10)\n    at Mongoose._promiseOrCallback (C:\\Users\\LENOVO\\Desktop\\typesript_npm\\node_modules\\mongoose\\lib\\index.js:1149:10)\n    at Mongoose.connect (C:\\Users\\LENOVO\\Desktop\\typesript_npm\\node_modules\\mongoose\\lib\\index.js:350:20)\n    at Object.connect (C:\\Users\\LENOVO\\Desktop\\typesript_npm\\src\\database\\connection.ts:8:12)\n    at Server.<anonymous> (C:\\Users\\LENOVO\\Desktop\\typesript_npm\\src\\app.ts:21:10)\n    at Object.onceWrapper (events.js:421:28)\n    at Server.emit (events.js:315:20)\n    at Server.EventEmitter.emit (domain.js:467:12)\n    at emitListeningNT (net.js:1352:10)\n    at processTicksAndRejections (internal/process/task_queues.js:79:21)","timestamp":"2021-06-21T12:22:29.896Z"}
{"message":"error ....in create user","level":"error","timestamp":"2021-06-21T12:33:00.885Z"}
{"code":"ETIMEOUT","syscall":"querySrv","hostname":"_mongodb._tcp.cluster0.4yiea.mongodb.net","level":"error","message":"db error querySrv ETIMEOUT _mongodb._tcp.cluster0.4yiea.mongodb.net","stack":"Error: querySrv ETIMEOUT _mongodb._tcp.cluster0.4yiea.mongodb.net\n    at QueryReqWrap.onresolve [as oncomplete] (dns.js:203:19)","timestamp":"2021-06-22T04:18:32.574Z"}
{"errors":{"password":{"name":"ValidatorError","message":"Path `password` is required.","properties":{"message":"Path `password` is required.","type":"required","path":"password"},"kind":"required","path":"password"},"phonenumber":{"name":"ValidatorError","message":"Path `phonenumber` is required.","properties":{"message":"Path `phonenumber` is required.","type":"required","path":"phonenumber"},"kind":"required","path":"phonenumber"},"email":{"name":"ValidatorError","message":"Path `email` is required.","properties":{"message":"Path `email` is required.","type":"required","path":"email"},"kind":"required","path":"email"},"address":{"name":"ValidatorError","message":"Path `address` is required.","properties":{"message":"Path `address` is required.","type":"required","path":"address"},"kind":"required","path":"address"},"name":{"name":"ValidatorError","message":"Path `name` is required.","properties":{"message":"Path `name` is required.","type":"required","path":"name"},"kind":"required","path":"name"}},"_message":"User validation failed","level":"error","timestamp":"2021-06-22T07:21:00.542Z"}
{"message":"error in image controller","level":"error","timestamp":"2021-06-22T08:41:28.089Z"}
{"message":"error in image controller","level":"error","timestamp":"2021-06-22T08:41:56.158Z"}
{"message":"error ....in create user","level":"error","timestamp":"2021-06-22T09:37:39.168Z"}
{"message":"error ....in create user","level":"error","timestamp":"2021-06-22T09:38:23.768Z"}
{"message":"error ....in create user","level":"error","timestamp":"2021-06-22T09:38:32.228Z"}
